### ISSUE TITLE
v0.106.3 can't access remote server

### ISSUE THREAD

**Issue by dreamwell0** [microsoft/vscode-remote-release#8958]
v0.106.3 can't access remote server .
There are logs.
It is occurring not only on Windows but also on Mac.

*v0.105.1 can access remote server.

```
[11:47:34.421] Install and start server if needed
[11:47:34.428] Checking ssh with "C:\Windows\system32\ssh.exe -V"
[11:47:34.431] Got error from ssh: spawn C:\Windows\system32\ssh.exe ENOENT
[11:47:34.432] Checking ssh with "C:\Windows\ssh.exe -V"
[11:47:34.432] Got error from ssh: spawn C:\Windows\ssh.exe ENOENT
[11:47:34.432] Checking ssh with "C:\Windows\System32\Wbem\ssh.exe -V"
[11:47:34.433] Got error from ssh: spawn C:\Windows\System32\Wbem\ssh.exe ENOENT
[11:47:34.433] Checking ssh with "C:\Windows\System32\WindowsPowerShell\v1.0\ssh.exe -V"
[11:47:34.434] Got error from ssh: spawn C:\Windows\System32\WindowsPowerShell\v1.0\ssh.exe ENOENT
[11:47:34.434] Checking ssh with "C:\Windows\System32\OpenSSH\ssh.exe -V"
・・・
[11:47:51.417] Failed to parse remote port from server output
[11:47:51.419] Resolver error: Error:
	at g.Create (c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:637876)
	at t.handleInstallOutput (c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:635279)
	at t.tryInstall (c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:756875)
	at async c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:717467
	at async t.withShowDetailsEvent (c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:720720)
	at async C (c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:714381)
	at async t.resolve (c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:718116)
	at async c:\Users\UserName\.vscode\extensions\ms-vscode-remote.remote-ssh-0.106.3\out\extension.js:2:901786
[11:47:51.429] ------
```

**Comment by mbrav** [comment]
Can confirm, had to roll back to `v0.105.1` from `v0.106.3` and everything worked fine.

**Comment by zeroflaw** [comment]
I found it I put the setting of 'Local Server Download' to always. it would start up correctly.

**Comment by fifv** [comment]
Same here.
I switched to pre-release 0.107.2023090815 and it works.
However, same problem occurs in the latest pre-release 0.107.2023091115

**Comment by Yapwn** [comment]
Same issue here. 
switched to o v0.105.1 from v0.107.2023091115 and it works.

**Comment by leohxj** [comment]
doesn't work with  0.106.3 and 0.107.2023091115

**Comment by JaroCamphuijsen** [comment]
Same issue here. Rollback to v0.105.1 solved it. 
But I would like to add that I had to remove the `.vscode-server` directory on the remote as well to make it work.

**Comment by iamashwin99** [comment]
same issue here.

**Comment by arigon** [comment]
Same here. How can I revert without deleting the folder?

**Comment by arigon** [comment]
> I found if I put the setting of 'Local Server Download' to always. it would start up correctly (v0.106.3).

This works for me!

**Comment by linderlake** [comment]
Had the same error, rebooting the server seemed to have solved it for me.

**Comment by roblourens** [comment]
The particular issue that I see here should be fixed in 0.106.4. If you are still having trouble, please open a new issue, thanks
